src/apps/opening-hours/OpeningHoursHelpers.ts

```diff
@@ -0,0 +1,80 @@
+import dayjs from "dayjs";
+import "dayjs/locale/da";
+import weekOfYear from "dayjs/plugin/weekOfYear";
+import { DplOpeningHoursListGET200Item as OpeningHoursItem } from "../../core/dpl-cms/model";
+
+dayjs.extend(weekOfYear);
+export const getNextWeek = (date: Date): Date => {
+  return dayjs(date).add(1, "week").toDate();
+};
+
+export const getPreviousWeek = (date: Date): Date => {
+  return dayjs(date).subtract(1, "week").toDate();
+};
+
+export const getWeekDisplay = (date: Date): string => {
+  const week = dayjs(date).week();
+  const year = dayjs(date).year();
+
+  return `Uge ${week}, ${year}`;
+};
+
+const capitalizeString = (string: string): string => {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+};
+
+export const formatDateToWeekday = (date: Date): string => {
+  const formattedAsWeekday = dayjs(date).locale("da").format("dddd");
+  const capitalizedWeekday = capitalizeString(formattedAsWeekday);
+
+  return capitalizedWeekday;
+};
+
+export const formatDate = (date: Date): string => {
+  return dayjs(date).locale("da").format("YYYY-MM-DD");
```

Comment:
Consider renaming or documenting the purpose of this function.

It is called `formatDate` where I initially assumed this would be used for formatting the date for end user display like `formatDateToWeekday()`. Having read the code it seams as if it is instead for formatting the date in ISO8601 format used by the API.